src/main/resources/templates/greeting.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE HTML>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>Getting Started: Serving Web Content</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+</head>
+<body>
+<p th:text="'Hello, ' + ${name} + '!'" />
+</body>
+</html>
```

Comment:
This file seems to be a duplicate of `build/resources/main/templates/greeting.html`. Which one are you using?

Reply:
Remember to delete the duplicated class :)